apps/ejabberd/src/mod_muc_light_admin.erl

```diff
@@ -119,9 +119,10 @@ create_unique_room(Domain, RoomName, Creator, Subject) ->
 
 invite_to_room(Domain, RoomName, Sender, Recipient) ->
     R = muc_light_room_name_to_jid(jid:from_binary(Sender), RoomName, Domain),
-    S = jid:from_binary(Sender),
+    S = jid:replace_resource(jid:from_binary(Sender), <<>>),
```

Comment:
Yes, I checked. @michalwski: We could wait for some time with the assumption that if it isn't received in the given time then it's never received.